share/instruments/instruments.xml

```diff
@@ -10558,20 +10520,20 @@
                   <pPitchRange>38-79</pPitchRange>
                   <singleNoteDynamics>0</singleNoteDynamics>
                   <Channel>
-                        <controller ctrl="32" value="8"/> <!--12-String Guitar-->
+                        <controller ctrl="32" value="8"/>
                         <program value="25"/> <!--Acoustic Guitar (steel)-->
                   </Channel>
                   <genre>earlymusic</genre>
             </Instrument>
             <Instrument id="ren.-tenor-lute-9-course">
+                  <trackName>Lute 9-course</trackName>
                   <longName>Lute</longName>
                   <shortName>Lt.</shortName>
-                  <trackName>Lute 9-course</trackName>
                   <description>Ren. Tenor Lute (9-course)</description>
                   <musicXMLid>pluck.lute</musicXMLid>
                   <StringData>
                         <frets>13</frets>
-                        <string open="1">36</string>
+                        <string>36</string>
```

Comment:
sure about this?

Reply:
No, we need to keep the `open` attribute.

Reply:
So a bug in the script, right?

Reply:
That's right. Also an omission from the spreadsheet.

Reply:
So at least my review wasn't pointless ;-)

Reply:
I had noticed it already, but it took me a week to get this far so I wanted to make a draft PR to show some progress.